.github/workflows/issue-analyzer.yml

```
@@ -1,44 +1,44 @@
-name: Issue Analyzer
+# name: Issue Analyzer
```

actionlint (1.7.4)

workflow is empty

(syntax-check)